coders/cut.c

```diff
@@ -384,7 +384,7 @@ static Image *ReadCUTImage(const ImageInfo *image_info,ExceptionInfo *exception)
       if (offset < 0)
         ThrowCUTReaderException(CorruptImageError,"ImproperImageHeader");
       if(EOFBlob(image) != MagickFalse) goto CUT_KO;  /*wrong data*/
-      EncodedByte=(size_t) ((ssize_t) EncodedByte-i+1);
+      EncodedByte=(size_t) ((ssize_t) EncodedByte-(i+1));
```

Comment:
Thanks for the fix but I am wondering if we could not write it like this instead:
```c
EncodedByte-=(size_t)(i+1);
```

Reply:
I'm assuming the double cast was added for a good reason, so I left it there. The IM6 code is simply EncodedByte-=i+1 and I can't see how that could overflow, so if you want to change it then that would be fine too. (As far as I can tell, both EncodedByte and i are guaranteed to be small.)

Reply:
Let's change it to `EncodedByte-=(size_t)(i+1);` for the readability.